google-cloud-testing/spanner-benchwrapper/src/main/java/com/google/cloud/benchwrapper/Main.java

```diff
@@ -20,7 +20,7 @@
 import io.grpc.netty.NettyServerBuilder;
 import java.util.Properties;
 
-class Main {
+public class Main {
```

Comment:
nit: I don't think you need to make this a public class.

Reply:
The lack of `public` is actually causing the benchmarking tests to fail according to the test logs:

```
java.lang.Exception: The specified mainClass doesn't contain a main method with appropriate signature.
    at org.codehaus.mojo.exec.ExecJavaMojo$1.run (ExecJavaMojo.java:258)
    at java.lang.Thread.run (Thread.java:748)
Caused by: java.lang.IllegalAccessException: symbolic reference class is not public: class com.google.cloud.benchwrapper.Main, from org.codehaus.mojo.exec.ExecJavaMojo$1
    at java.lang.invoke.MemberName.makeAccessException (MemberName.java:850)
    at java.lang.invoke.MethodHandles$Lookup.checkSymbolicClass (MethodHandles.java:1402)
    at java.lang.invoke.MethodHandles$Lookup.resolveOrFail (MethodHandles.java:1382)
    at java.lang.invoke.MethodHandles$Lookup.findStatic (MethodHandles.java:780)
    at org.codehaus.mojo.exec.ExecJavaMojo$1.run (ExecJavaMojo.java:251)
    at java.lang.Thread.run (Thread.java:748)
```

Let me know if there's a better way to fix this :-)

Reply:
Ah, ok. No, the easiest way is probably to just make it public.

(Note: This is not a Java requirement, but apparently a requirement for the Maven exec plugin. Java allows you to define the `main` method in a package-private class.)